docs-src/cli/workflow/list.md

```diff
@@ -11,10 +11,10 @@ tags:
   - workflow list
 ---
 
-The `temporal workflow list` command provides a list of [Workflow Executions](/concepts/what-is-a-workflow-execution) that meet the criteria of a given [Query](/concepts/what-is-a-query).
+The `temporal workflow list` command provides a list of [Workflow Executions](/concepts/what-is-a-workflow-execution) that meet the criteria of a given [List Filter](/visibility#list-filter).
```

Comment:
```suggestion
The `temporal workflow list` command provides a list of [Workflow Executions](/concepts/what-is-a-workflow-execution) that meet the criteria of a given [List Filter](/concepts/what-is-a-list-filter).
```

Just switching the link to use the id of the file - that way the link automatically updates if the target info moves or changes in anyway.